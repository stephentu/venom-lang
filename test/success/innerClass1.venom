class Outer
  attr x::int
  def self(x::int) = print(x); end
  class Inner
    attr x::int = 5
    def self(x::int) = self.x = x; end
  end
  def make() -> Inner = Inner(15); end
end
i::Outer.Inner = Outer(10).make();
j = Outer(20).make();
print(i.x);
print(j.x);
