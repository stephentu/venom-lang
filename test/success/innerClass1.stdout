10
20
15
15
